api/api.go

```diff
@@ -674,12 +679,12 @@ func (c *client) executeWithRetryAuthenticate(ctx context.Context, method, uri s
 
 func FetchValueIndexForKey(l string, match string, sep string) (int, int, int) {
 
+	var startIndex, endIndex = -1, -1
 	if strings.Contains(l, match) {
-		if i := strings.Index(l, match); i != -1 {
-			if j := strings.Index(l[i+len(match):], sep); j != -1 {
-				return i, j, len(match)
-			}
+		startIndex = strings.Index(l, match)
+		if sep != "" {
```

Comment:
could do  startIndex != -1  && sep != "" to avoidgoing inside the if to run the strings operation

Reply:
Updated